README.md
# otolithe
Otolith is an application which allows you to place points referenced XY on photos to calculate specific indicators. It was designed specifically to analyze specific parts of fish, such as otoliths (ear bones), but can also be used to treat scaly, or possibly also of fin rays.

The code of version 2.0 has been desposited from Agence pour la Protection des Programmes under the number IDDN.FR.001.440010.000.S.C.2018.000.20900

For install this software:
- install a linux server (Ubuntu or Debian)
- download install/deploy_new_instance.sh
- verify the content of the script, particulary the variable phpinifile (version of php). It may be 7.2 for Ubuntu 18.04.
- execute this script : it will install all necessary packages, create the database, etc.
- edit the file /etc/apache2/sites-available/otolithe.conf and adapt it to your configuration
- reload apache server

A manual of installation and usage is available in french at https://github.com/Irstea/otolithe/raw/master/database/documentation/otolithe_documentation.pdf


param/news.txt
Nouveautés
===========
Version 2.2.1 du 22 février 2019
---------------------------
Nouveautés :
- dans les tableaux, le nombre de lignes sélectionné est maintenant conservé
- de nouveaux critères de tri ont été introduits : espèce, lecture consensuelle, lecture non réalisée

Version 2.2 du 10 janvier 2019
---------------------------
Corrections de bugs :
- la suppression d'une pièce n'aboutissait pas

Nouveautés :
- dans les tableaux, lorsqu'un lien a été cliqué, celui-ci change de couleur (ancien fonctionnement, présent dans les versions 1.x)
- ajout d'une gestion de méta-données : il est possible d'ajouter des informations techniques à une pièce, soit sous forme unique (valeur unique pour un champ), soit sous la forme d'un tableau (multi-valeurs). Les données sont décrites sous forme de méta-données et stockées dans un champ JSON (export possible en CSV)

Version 2.1 du 13 décembre 2018
---------------------------
Corrections de bugs :
- lors de l'importation des poissons, les dates étaient mal gérées
- il n'était pas possible d'exporter une seule lecture
- lors de la saisie d'un nouveau poisson, la liste des expérimentations auxquelles il pouvait être rattaché n'était pas affichée
- il n'était pas possible pour un lecteur de supprimer une lecture qu'il avait effectuée

Nouveautés :
- il est possible de rajouter un commentaire textuel lors d'une lecture
- lors de la lecture, il est possible de définir certains points comme "remarquables", par exemple pour signaler des stries de croissance particulières
- ajout d'un icone permettant de supprimer un point depuis la table récapitulant les points positionnés : cela permet la suppression d'un point non seulement par double-clic sur celui-ci, mais également depuis la liste quand le double-clic n'est pas utilisable (saisie depuis un smartphone, notamment)

Version 2.0 du 18 août 2018
---------------------------
Refonte de l'interface et intégration de la dernière version du framework
Nouveautés et améliorations :
- il est possible de modifier directement une espèce ou en rajouter une
- gestion des photos TIFF : il est possible de télécharger une photo au format TIFF. Toutefois, la saisie des points se fait obligatoirement avec une photo au format JPG (transformation dynamique). Quelques limitations à l'utilisation des photos TIFF :
> Elles doivent être "propres", c'est à dire sans anomalies (les appareils de laboratoire peuvent générer des fichiers mal formatés)
> leur taille est volontairement limitée à 50 Mo. Au delà, des problèmes de performance au niveau du serveur peuvent être ressenties, voire bloquer totalement le serveur.
- les données affichées dans le détail d'un poisson ont été volontairement réduites, pour ne pas "polluer" toutes les expérimentations avec des données spécifiques à certaines
- l'importation de données permet de renseigner tous les champs gérés par l'application pour les poissons. Un fichier CSV vierge peut être téléchargé
- il est maintenant possible de renseigner directement les lecteurs depuis le détail d'une expérimentation

Correction de bugs
- si l'opérateur qui crée une expérimentation est indiqué comme lecteur de celle-ci, il récupère les droits automatiquement sans être obligé de se reconnecter

Version 1.5.1 du 11 octobre 2017
------------------------------
Extension de la fonction d'import automatique, pour rajouter des informations complémentaires (besoin exprimé pour des lectures de rayons d'esturgeons)

Version 1.5 du 16 février 2017
------------------------------
Modifications concernant la lecture des photos :
- si tous les points ont été supprimés, il est maintenant possible de reprendre la numérotation à 0
- si le dernier point a été supprimé, le compteur est décrémenté de 1 (le point suivant aura la bonne valeur)
- lors de l'affichage des lectures, le cercle est maintenant opaque. La résolution et le facteur de transparence sont également conservés d'une photo à l'autre
- rajout de 2 résolutions pour la lecture ou l'affichage, dont une correspondant à la taille (en principe) initiale de la photo (10000x10000 pixels)
- la photo affichée ne peut pas être plus grande que la photo téléchargée (c'était possible avant)

Modifications concernant la gestion des paramètres et l'import automatique d'individus
- une nouvelle expérimentation peut être rajoutée (droit nécessaire : gestionCompte)
- un nouveau type de pièce peut être rajouté (droit nécessaire : gestionCompte)
- il est maintenant possible d'importer automatiquement des individus et la référence de la pièce calcifiée à analyser


Version 1.4 du 3 décembre 2015
------------------------------
Améliorations générales :
- en ajout d'une photo, le programme propose la lumière incidente par défaut. De plus, les nouvelles photos sont analysées avec un antivirus avant enregistrement dans la base
- les poissons ou les lectures sont affichés par page de 100
- la date de pêche du poisson est maintenant indiquée dans le cartouche

Données rajoutées pour compléter la lecture des pièces (positionnement des points) :
- indicateur permettant de préciser s'il s'agit d'une lecture consensuelle
- année estimée de naissance (à calculer manuellement à partir du nombre de points positionnés et de la date de pêche)
- nature de la strie finale (hyaline, ombrée, non définie, ou vide)
- coefficient de fiabilité de la lecture, estimé par le lecteur (0, 0.5, 1 ou vide)

Ces 3 dernières informations ne sont pas affichées par défaut lors de la consultation d'une photo (tableau des lectures)

Corrections de bugs :
- suite aux modifications de juin, et en particulier le renforcement de la sécurité globale de l'application, il n'était plus possible de modifier une lecture en en consultant d'autres 

Version 1.3 du 15 juin 2015
---------------------------
Améliorations :
- il est maintenant possible de rajouter un poisson (et la pêche associée) à une expérimentation
- les lecteurs ne peuvent accéder qu'aux expérimentations pour lesquels ils ont été autorisés

locales/fr.php
<?php
define("FORMATDATE","fr");
$LANG= array();
$LANG["menu"][0] = "Gestion";
$LANG["menu"][1] = "Opérations de gestion";
$LANG["menu"][2] = "Liste des comptes";
$LANG["menu"][3] = "Liste des logins - identification via la base de données";
$LANG["menu"][4] = "Administration";
$LANG["menu"][5] = "Administration de l'application";
$LANG["menu"][6] = "Déconnexion";
$LANG["menu"][7] = "Déconnexion de l'application";
$LANG["menu"][8] = "A propos";
$LANG["menu"][9] = "A propos d'Otolithe";
$LANG["menu"][10] = "Aide";
$LANG["menu"][11] = "Quelques conseils...";
$LANG["menu"][12] = "Gestion des droits";
$LANG["menu"][13] = "Gestion des droits d'accès aux modules de l'application";
$LANG["menu"][20] = "Aide";
$LANG["menu"][21] = "Utiliser Otolithe";
$LANG["menu"][22] = "Installation";
$LANG["menu"][23] = "Installer PrototypePHP";
$LANG["menu"][24] = "Gestion des droits";
$LANG["menu"][25] = "Gérer les droits dans l'application";
$LANG["menu"][26] = "Module de lecture-écriture";
$LANG["menu"][27] = "Créer un module pour lire ou écrire un enregistrement";
$LANG["menu"][28] = "Paramétrage du module";
$LANG["menu"][29] = "Configurer le module dans le fichier xml";
$LANG["menu"][30] = "Classe d'accès aux données";
$LANG["menu"][31] = "Créer la classe permettant d'accéder à une table";
$LANG["menu"][32] = "Gestion de la documentation";
$LANG["menu"][33] = "Gérer les pages de la documentation";
$LANG["menu"][34] = "Langues";
$LANG["menu"][35] = "Gestion des différentes langues, internationalisation de l'application";
$LANG["menu"][36] = "SMARTY";
$LANG["menu"][37] = "Manipuler les templates SMARTY";
$LANG["menu"][38] = "Structure";
$LANG["menu"][39] = "Organisation de PrototypePHP";
$LANG["menu"][40] = "Nouveautés";
$LANG["menu"][41] = "Quoi de neuf dans l'application ?";
$LANG["menu"][42] = "Module de recherche";
$LANG["menu"][43] = "Intégrer un module de recherche";
$LANG["menu"][44] = "PHP info";
$LANG["menu"][45] = "Informations sur l'implémentation de PHP";
$LANG["menu"][50] = "Liste des poissons";
$LANG["menu"][51] = "Rechercher les poissons capturés...";
$LANG["menu"][52] = "Liste des lecteurs";
$LANG["menu"][53] = "Personnes susceptible de réaliser des lectures sur les pièces...";
$LANG["menu"][54] = "Liste des lectures";
$LANG["menu"][55] = "Afficher la liste les lectures de pièces...";
$LANG["menu"][56] = "Expérimentations";
$LANG["menu"][57] = "Liste des expérimentations";
$LANG["menu"][58] = "Types de pièces";
$LANG["menu"][59] = "Liste des types de pièces calcifiées";
$LANG["menu"][60] = "Paramètres";
$LANG["menu"][61] = "Tables de paramètres";
$LANG["menu"][62] = "Espèces";
$LANG["menu"][63] = "Liste des espèces";
$LANG["menu"][64] = "Import";
$LANG["menu"][65] = "Import d'individus associés ou non à des pièces";
$LANG["menu"][66] = "";
$LANG["menu"][67] = "";



/*
 * Gestion des droits nouvelle formule
 */
$LANG["menu"]["appliliste"] = "ACL - droits";
$LANG["menu"]["applilistetitle"] = "applications et droits gérés";
$LANG["menu"]["aclloginliste"] = "ACL - logins";
$LANG["menu"]["aclloginlistetitle"] = "Logins déclarés dans le module de gestion des droits";
$LANG["menu"]["groupliste"] = "ACL - groupes de logins";
$LANG["menu"]["grouplistetitle"] = "Groupes de logins et logins rattachés aux groupes";

$LANG["menu"]["example1"] = "Example";
$LANG["menu"]["example2"] = "It's an example !";
$LANG["menu"]["connexionvalue"] = "Connexion";
$LANG["menu"]['connexiontitle'] = "S'identifier dans l'application";

$LANG["message"][0] = "Bienvenue"; 
$LANG["message"][1] = "Lecture des otolithes et pièces calcifiées";
$LANG["message"][2] = "Module administration";
$LANG["message"][3] = "Module gestion";
$LANG["message"][4] = "Suppression effectuée";
$LANG["message"][5] = "Enregistrement effectué";
$LANG["message"][6] = "Vidage effectué";
$LANG["message"][7] = "Vous êtes maintenant déconnecté";
$LANG["message"][8] = "Vous n'êtes pas connecté";
$LANG["message"][9] = "Vous n'avez pas le droit d'accéder à cette fonction";
$LANG["message"][10] = "Identification réussie !";
$LANG["message"][11] = "Identification refusée";
$LANG["message"][12] = "Problème lors de la mise en fichier...";
$LANG["message"][13] = "Problème lors de la suppression";
$LANG["message"][14] = "Vous ne pouvez pas accéder directement à cette page";
$LANG["message"][15] = "oui";
$LANG["message"][16] = "non";
$LANG["message"][17] = "Attention :";
$LANG["message"][18] = "Confirmez-vous l'opération ?";
$LANG["message"][19] = "Valider";
$LANG["message"][20] = "Supprimer";
$LANG["message"][21] = "Rechercher";
$LANG["message"][22] = "Echec de connexion à la base de données principale";
$LANG["message"][23] = "Copyright (c) 2013-2014 - Tous droits réservés.";
$LANG["message"][24] = "Propriété de l'IRSTEA de Bordeaux-CESTAS";
$LANG["message"][25] = "Pour tout problème : ";
$LANG["message"][26] = "Vous n'avez pas accès à cette fonction. Veuillez vous identifier, le cas échéant...";
$LANG["message"][27] = "Vous n'êtes pas identifié.";
$LANG["message"][28] = "L'accès au module que vous demandez ne respecte pas la cinématique de l'application. 
		Peut-être avez-vous essayé de rafraîchir la page ?";
$LANG["message"][29] = "Echec de connexion à la base de données de gestion des droits (GACL)";
$LANG["message"][30] = "Champ obligatoire";
$LANG["message"][31] = "Confirmez-vous la suppression ?";
$LANG["message"][32] = "Mode développement - base de données";
$LANG["message"][33] = "Les modifications apportées dans cette page vont être perdues. Confirmez-vous cette opération ?";
$LANG["message"]["yes"] = "oui";
$LANG["message"]["no"] = "non";
$LANG["message"][34] = "Consulter";
$LANG["message"][35] = "Retour";
$LANG["message"][36] = "Ouvrez un ticket dans la forge";
$LANG["message"][37] = "Envoyez un message à l'assistance";
$LANG["message"]["ou"] = "ou";


$LANG["login"][0] = "Login";
$LANG["login"][1] = "Mot de passe";
$LANG["login"][2] = "Veuillez utiliser votre login du domaine pour vous identifier";
$LANG["login"][5] = "Nouveau login";
$LANG["login"][6] = "login";
$LANG["login"][7] = "Nom - prénom";
$LANG["login"][8] = "Mèl";
$LANG["login"][9] = "Nom";
$LANG["login"][10] = "Prénom";
$LANG["login"][11] = "Date";
$LANG["login"][12] = "Répétez le mot de passe";
$LANG["login"][13] = "actif";
$LANG["login"][14] = "Le mot de passe a déjà été utilisé";
$LANG["login"][15] = "Le mot de passe n'est pas assez complexe";
$LANG["login"][16] = "Le mot de passe est trop court";
$LANG["login"][17] = "Le mot de passe n'est pas identique dans les deux zones";
$LANG["login"][18] = "Le mode d'identification utilisé pour votre compte n'autorise pas la modification du mot de passe depuis cette application";
$LANG["login"][19] = "L'ancien mot de passe est incorrect";
$LANG["login"][20] = "Le mot de passe a été modifié";
$LANG["login"][21] = "Générez un mot de passe aléatoire";
$LANG["login"][22] = "Générez";
$LANG["login"][23] = "Ancien mot de passe";
$LANG["login"][24] = "Nouveau mot de passe";


$LANG["ObjetBDDError"][0] = "Le champ ";
$LANG["ObjetBDDError"][1] = " n'est pas numerique.";
$LANG["ObjetBDDError"][2] = " est trop grand. Longueur maximale autorisée : ";
$LANG["ObjetBDDError"][3] = " Valeur saisie : ";
$LANG["ObjetBDDError"][4] = " caractères";
$LANG["ObjetBDDError"][5] = "Le contenu du champ ";
$LANG["ObjetBDDError"][6] = " ne correspond pas au format attendu. Masque autorisé : ";
$LANG["ObjetBDDError"][7] = " est obligatoire, mais n'a pas été renseigné.";

$LANG["gestion"][0] = "Retour à la liste";
$LANG["gestion"][1] = "Code";
$LANG["gestion"][2] = "Tag";
$LANG["gestion"][3] = "Affichage du détail d'un individu";
$LANG["gestion"][4] = "Données générales";
$LANG["gestion"][5] = "Espèce";
$LANG["gestion"][6] = "Sexe";
$LANG["gestion"][7] = "Code de l'individu";
$LANG["gestion"][8] = "Longueur";
$LANG["gestion"][9] = "Poids";
$LANG["gestion"][10] = "Remarque";
$LANG["gestion"][11] = "Parasite";
$LANG["gestion"][12] = "Age";
$LANG["gestion"][13] = "Pectorale gauche";
$LANG["gestion"][14] = "Diamètre oculaire";
$LANG["gestion"][15] = "horizontal";
$LANG["gestion"][16] = "vertical";
$LANG["gestion"][17] = "Ht mm";
$LANG["gestion"][18] = "Épaisseur";
$LANG["gestion"][19] = "Circonférence";
$LANG["gestion"][20] = "Pièces rattachées";
$LANG["gestion"][21] = "Nouvelle pièce";
$LANG["gestion"][22] = "Type";
$LANG["gestion"][23] = "Traitement réalisé";
$LANG["gestion"][24] = "Nbre photos rattachées";
$LANG["gestion"][25] = "Expérimentation(s)";
$LANG["gestion"][26] = "Données concernant la pêche";
$LANG["gestion"][27] = "Site";
$LANG["gestion"][28] = "Zone précise";
$LANG["gestion"][29] = "Date de pêche";
$LANG["gestion"][30] = "Campagne";
$LANG["gestion"][31] = "Engin";
$LANG["gestion"][32] = "Pêcheur";
$LANG["gestion"][33] = "Opérateur";
$LANG["gestion"][34] = "Données physico-chimiques de la pêche";
$LANG["gestion"][35] = "Analyse";
$LANG["gestion"][36] = "Valeur mesurée";
$LANG["gestion"][37] = "Valeur mini";
$LANG["gestion"][38] = "Valeur maxi";
$LANG["gestion"][39] = "Température";
$LANG["gestion"][40] = "O2";
$LANG["gestion"][41] = "pourcentage O2";
$LANG["gestion"][42] = "Conductivité";
$LANG["gestion"][43] = "Salinité";
$LANG["gestion"][44] = "pH";
$LANG["gestion"][45] = "Code individu";
$LANG["gestion"][46] = "Nbre de";
$LANG["gestion"][47] = "pièces";
$LANG["gestion"][48] = "Zone de pêche";
$LANG["gestion"][49] = "Expérimentation";
$LANG["gestion"][50] = "Code ou TAG de l'individu";
$LANG["gestion"][51] = "Sexe de l'animal";
$LANG["gestion"][52] = "Sélectionnez le sexe recherché";
$LANG["gestion"][53] = "Sélectionnez l'expérimentation concernée";
$LANG["gestion"][54] = "Site de pêche";
$LANG["gestion"][55] = "Sélectionnez le site global de pêche";
$LANG["gestion"][56] = "Zone précise de pêche";
$LANG["gestion"][57] = "Sélectionnez le site précis de pêche";
$LANG["gestion"][58] = "Login de connexion";
$LANG["gestion"][59] = "le nom est obligatoire";
$LANG["gestion"][60] = "le login est obligatoire";
$LANG["gestion"][61] = "Liste des lecteurs potentiels";
$LANG["gestion"][62] = "Nouveau";
$LANG["gestion"][63] = "Modification d'une photo";
$LANG["gestion"][64] = "Retour au détail du poisson";
$LANG["gestion"][65] = "Retour au détail de la pièce";
$LANG["gestion"][66] = "Nom de la photo";
$LANG["gestion"][67] = "Description";
$LANG["gestion"][68] = "Fichier JPG contenant la photo";
$LANG["gestion"][69] = "Nom du fichier";
$LANG["gestion"][70] = "Date de prise de vue";
$LANG["gestion"][71] = "Couleur";
$LANG["gestion"][72] = "Noir et blanc";
$LANG["gestion"][73] = "Type de lumière";
$LANG["gestion"][74] = "Grossissement";
$LANG["gestion"][75] = "Repère";
$LANG["gestion"][76] = "Adresse de stockage (fichiers dans arborescence)";
$LANG["gestion"][77] = "Repère de mesure - longueur de référence";
$LANG["gestion"][78] = "Dimensions de la photo";
$LANG["gestion"][79] = "Affichage d'une photo";
$LANG["gestion"][80] = "URI";
$LANG["gestion"][81] = "Attention : le temps de chargement peut être (très) long, selon la taille originale de la photo !";
$LANG["gestion"][82] = "Résolution (approximative) de lecture";
$LANG["gestion"][83] = "Réaliser une nouvelle lecture";
$LANG["gestion"][84] = "Lecteur";
$LANG["gestion"][85] = "Date de lecture";
$LANG["gestion"][86] = "Résolution";
$LANG["gestion"][87] = "Age ou nb de segments<br>positionnés (nb points - 1)";
$LANG["gestion"][88] = "Afficher l'age calculé (nbre de points positionnés - 1) par chaque lecteur";
$LANG["gestion"][89] = "Longueur de";
$LANG["gestion"][90] = "référence mesurée";
$LANG["gestion"][91] = "Longueur totale lue";
$LANG["gestion"][92] = "Longueur réelle";
$LANG["gestion"][93] = "calculée";
$LANG["gestion"][94] = "Résolution (approximative) d'affichage";
$LANG["gestion"][95] = "Déclencher l'affichage des lectures sélectionnées, avec ou sans création/modification d'une lecture";
$LANG["gestion"][96] = "Patientez pendant le chargement";
$LANG["gestion"][97] = "Modification d'une pièce";
$LANG["gestion"][98] = "Type de pièce";
$LANG["gestion"][99] = "Code de la pièce";
$LANG["gestion"][100] = "Traitement effectué";
$LANG["gestion"][101] = "Affichage d'une pièce";
$LANG["gestion"][102] = "Détail de la pièce";
$LANG["gestion"][103] = "Photos rattachées";
$LANG["gestion"][104] = "Nouvelle photo";
$LANG["gestion"][105] = "Nom";
$LANG["gestion"][106] = "Date";
$LANG["gestion"][107] = "Dimensions";
$LANG["gestion"][108] = "Miniature";
$LANG["gestion"][109] = "Sélectionnez le lecteur";
$LANG["gestion"][110] = "Mesure d'un otolithe ou d'une pièce calcifiée";
$LANG["gestion"][111] = "Retour à la photo";
$LANG["gestion"][112] = "Enregistrez les points positionnés";
$LANG["gestion"][113] = "Type de lecture pour le prochain point";
$LANG["gestion"][114] = "Point initial avec cercle élargi";
$LANG["gestion"][115] = "Lecture des points";
$LANG["gestion"][116] = "Mesure de la longueur de référence";
$LANG["gestion"][117] = "Tracé d'une ligne sur la photo (aide à la mesure)";
$LANG["gestion"][118] = "Taille originale de la photo";
$LANG["gestion"][119] = "Taille de lecture de la photo";
$LANG["gestion"][120] = "Coefficient de correction de la taille";
$LANG["gestion"][121] = "Rayon (en pixels) du cercle élargi";
$LANG["gestion"][122] = "Recalcul automatique de l'ordre des points";
$LANG["gestion"][123] = "Points sélectionnés";
$LANG["gestion"][124] = "N°";
$LANG["gestion"][125] = "Ordre de";
$LANG["gestion"][126] = "lecture";
$LANG["gestion"][127] = "Point de mesure";
$LANG["gestion"][128] = "de la longueur";
$LANG["gestion"][129] = "de référence";
$LANG["gestion"][130] = "Pour supprimer un point, réalisez un double-clic sur celui-ci";
$LANG["gestion"][131] = "Vous pouvez modifier manuellement l'ordre de lecture d'un point, si nécessaire";
$LANG["gestion"][132] = "Pour tracer une ligne, positionnez le premier point, puis le second. Pour supprimer la ligne, supprimez d'abord le second point, avant de toucher au premier point.";
$LANG["gestion"][133] = "Le recalcul automatique de l'ordre des points ne fonctionne que si le premier point (origine) est saisi en premier (valeur ".'"ordre de lecture" la plus faible de la série).';
$LANG["gestion"][134] = "Affichage des mesures d'un otolithe";
$LANG["gestion"][135] = "Résolution d'affichage";
$LANG["gestion"][136] = "Légende";
$LANG["gestion"][137] = "Consultation de la lecture de pièces";
$LANG["gestion"][138] = "Individu";
$LANG["gestion"][139] = "Pièce";
$LANG["gestion"][140] = "traitement";
$LANG["gestion"][141] = "Photo";
$LANG["gestion"][142] = "Couleur/NB";
$LANG["gestion"][143] = "Résolution<br>de lecture";
$LANG["gestion"][144] = "Résolution<br>de prise de vue";
$LANG["gestion"][145] = "Date de<br>prise de vue";
$LANG["gestion"][146] = "Longueur de<br>référence";
$LANG["gestion"][147] = "Lecteur";
$LANG["gestion"][148] = "Date de lecture";
$LANG["gestion"][149] = "Longueur de<br>référence mesurée";
$LANG["gestion"][150] = "Longueur réelle totale<br>de la mesure";
$LANG["gestion"][151] = "lecture du";
$LANG["gestion"][152] = "Séparateur : tabulation, UTF-8, chiffres décimaux avec points";
$LANG["gestion"][153] = "Exporter la liste au format CSV";
$LANG["gestion"][154] = "Modifier la photo";
$LANG["gestion"][155] = "Modif<br>unique";
$LANG["gestion"][156] = "Consulter";
$LANG["gestion"][157] = "Facteur de transparence des cercles affichés";
$LANG["gestion"][158] = "Transparent";
$LANG["gestion"][159] = "Légèrement ombré";
$LANG["gestion"][160] = "Ombré";
$LANG["gestion"][161] = "Semi-opaque";
$LANG["gestion"][162] = "Opaque";
$LANG["gestion"][163] = "Création d'une nouvelle lecture simple";
$LANG["gestion"][164] = "Consultations individuelles, globales, modifications avec visualisation des points déjà tracés";
$LANG["gestion"][165] = "Lecture à<br>modifier";
$LANG["gestion"][166] = "Avec création d'une nouvelle lecture";
$LANG["gestion"][167] = "Si coché, la lecture sélectionnée pourra être modifiée";
$LANG["gestion"][168] = "Taille en pixels de la longueur de référence dans la photo";
$LANG["gestion"][169] = "Masquer l'age calculé (nbre de points positionnés - 1) par chaque lecteur";
$LANG["gestion"][170] = "Nature de la strie finale";
$LANG["gestion"][171] = "Strie finale";
$LANG["gestion"][172] = "Fiabilité de la lecture";
$LANG["gestion"][173] = "Lecture consensuelle";
$LANG["gestion"][174] = "Année de naissance estimée";
$LANG["gestion"][175] = "Année de naissance";




?>
